api/types/volume/cluster_volume.go

```diff
@@ -104,7 +104,7 @@ type AccessMode struct {
 	BlockVolume *TypeBlock `json:",omitempty"`
 }
 
-// Scope defines the Scope of a CSI Volume. This is how many nodes a
+// Scope defines the Scope of a Cluster Volume. This is how many nodes a
```

Comment:
e.g.; these; wasn't sure where we wanted to use the "Cluster" terminology and where we would still have to refer to "CSI"